src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkMigrationService.kt

```diff
@@ -176,6 +182,10 @@ class VideoLinkMigrationService(
     )
   }
 
+  private fun VideoLinkBooking.isProbationBooking() =
+    courtName?.uppercase()?.contains("PROBATION") == true ||
+      courtService.getCourtNameForCourtId(courtId)?.uppercase()?.contains("PROBATION") == true
```

Comment:
It might pay to have a little peak at this service method.

This also handles the situation where the `court id == 'other'` on bookings to migrate.  There is no court for `other`.